sailpoint.sgnl.yaml

```
@@ -0,0 +1,1425 @@
+        type: String
+      - name: identityValue
+        externalId: $.identity.value
+        type: String
+      - name: identityRef
+        externalId: $.identity..["$ref"]
+        type: String
+      - name: identityDisplayName
+        externalId: $.identity.displayName
+        type: String
+      - name: ownerValue
+        externalId: $.owner.value
+        type: String
+      - name: ownerRef
+        externalId: $.owner..["$ref"]
+        type: String
+      - name: ownerDisplayName
+        externalId: $.owner.displayName
+        type: String
+      - name: description
+        externalId: description
+        type: String
+      - name: status
+        externalId: status
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  Role:
+    displayName: SailpointRole
+    externalId: Roles
+    description: Entity representing a Sailpoint Identity IQ Role.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        uniqueId: true
+        indexed: true
+      - name: name
+        externalId: name
+        type: String
+      - name: typeIIQ
+        externalId: $.type.iiq
+        type: Bool
+      - name: typeRequirements
+        externalId: $.type.requirements
+        type: Bool
+      - name: typePermits
+        externalId: $.type.permits
+        type: Bool
+      - name: typeDisplayName
+        externalId: $.type.displayName
+        type: String
+      - name: typeManualAssignment
+        externalId: $.type.manualAssignment
+        type: Bool
+      - name: typeName
+        externalId: $.type.name
+        type: String
+      - name: typeAutoAssignment
+        externalId: $.type.autoAssignment
+        type: Bool
+      - name: typeAssignmentSelector
+        externalId: $.type.assignmentSelector
+        type: Bool
+      - name: displayableName
+        externalId: displayableName
+        type: String
+      - name: active
+        externalId: active
+        type: Bool
+      - name: activationDate
+        externalId: activationDate
+        type: DateTime
+      - name: deactivationDate
+        externalId: deactivationDate
+        type: DateTime
+      - name: ownerDisplayName
+        externalId: $.owner.displayName
+        type: String
+      - name: ownerValue
+        externalId: $.owner.value
+        type: String
+      - name: ownerRef
+        externalId: $.owner..$.["$ref"]
```

This looks to technically work, but is inconsistent with every other instance
```suggestion
        externalId: $.owner..["$ref"]
```

---

You changed this to `$.owner..$..["$ref"]` instead of `$.owner..["$ref"]` which missed the point of my comment. Why are you adding the additional `$.`/`$..`? There are 10 other instances of accessing a nested `$ref` field all without the second `$..` - is there something special about this case that warrants the inconsistent handling I'm not aware of?

---

I misread the comment initially. Fixed it now.